docs/configuration-with-ocis.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Running"
+date: 2018-05-02T00:00:00+00:00
+weight: 50
+geekdocRepo: https://github.com/owncloud/ocis-hello
+geekdocEditPath: edit/master/docs
+geekdocFilePath: configuration-with-ocis.md
+---
+
+### Configuring ocis-hello with ocis
+We will need various services to run ocis
+
+#### Running a ldap server in docker container
+We will use the ldap server as users provider for ocis.
+```
+docker run --hostname ldap.my-company.com \
+    -e LDAP_TLS_VERIFY_CLIENT=never \
+    -e LDAP_DOMAIN=owncloud.com \
+    -e LDAP_ORGANISATION=ownCloud \
+    -e LDAP_ADMIN_PASSWORD=admin \
+    --name docker-slapd \
+    -p 127.0.0.1:389:389 \
+    -p 636:636 -d osixia/openldap
+```
+#### Running a redis server in a docker container
+Redis will be used by ocis form various caching purposes.
+```
+docker run -e REDIS_DATABASES=1 -p 6379:6379 -d webhippie/redis:latest
+
+```
+#### Running ocis
+In order to run this extension we will need to run ocis first. For that clone and build the ocis single binary from the github repo `https://github.com/owncloud/ocis`.
+After that we will need to create a config file for phoenix so that we can load the hello app in the frontend. Create a file `phoenix-config.json` with following contents.
```

Comment:
```suggestion
After that we will need to create a config file for phoenix so that we can load the hello app in the frontend. Create a file `phoenix-config.json` with the following contents.
```